lib/jsdom/browser/Window.js

```diff
@@ -60,15 +57,27 @@ function Window(options) {
   ///// INTERFACES FROM THE DOM
   // TODO: consider a mode of some sort where these are not shared between all DOM instances
   // It'd be very memory-expensive in most cases, though.
-  for (const name in dom) {
+  this._core = bootstrapDOM("Window", window, { window });
```

Comment:
We no longer store `_core` on Window after #2098. Not sure how that fits in here.